app/controllers/katello/api/v2/package_groups_controller.rb

```diff
@@ -2,11 +2,28 @@ module Katello
   class Api::V2::PackageGroupsController < Api::V2::ApiController
     apipie_concern_subst(:a_resource => N_("a package group"), :resource => "package_groups")
     include Katello::Concerns::Api::V2::RepositoryContentController
+    include Katello::Concerns::Api::V2::RepositoryDbContentController
+
+    api :GET, "package_groups/:id", N_("show package groups by id")
+    def index
+      super
+    end
+
+    def available_for_content_view_filter(filter, collection)
+      collection_ids = []
+      current_ids = filter.package_group_rules.map(&:uuid)
+      filter.applicable_repos.each do |repo|
+        collection_ids.concat(repo.package_groups.map(&:uuid))
+      end
+      collection = PackageGroup.where(:uuid => collection_ids)
+      collection = collection.where("uuid not in (?)", current_ids) unless current_ids.empty?
+      collection
+    end
 
     private
 
     def repo_association
-      :repo_id
+      :repository_id
     end
   end
 end
```

Comment:
You probably need to set a default sort here similar to:

https://github.com/Katello/katello/blob/master/app/controllers/katello/api/v2/errata_controller.rb#L59

but with 'name asc'